website_libs/dev_deps.js

```diff
@@ -201,4 +201,48 @@
     for (var j = 0, lenj = dismissList.length; j < lenj; j++) {
         dismissList[j].onclick = doDismiss;
     }
+    
```

Comment:
dev_deps.js is pulled into example code only and it is never used as part of the library.